osmoutils/generic_helper.go

```diff
@@ -15,3 +19,17 @@ func MakeNew[T any]() T {
 		return *new(T) // v is not ptr, alloc with new
 	}
 }
+
+func Min[T constraints.Ordered](a, b T) T {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func Max[T constraints.Ordered](a, b T) T {
+	if a > b {
```

Comment:
```suggestion
	if a >= b {
```

If the two values are equal, we should preserve the input order

Reply:
shouldn't it be like this in the Min case then?